service/src/test/java/com/ge/predix/acs/attribute/connectors/DefaultResourceAttributeStoreTest.java

```diff
@@ -0,0 +1,33 @@
+package com.ge.predix.acs.attribute.connectors;
+
+import com.ge.predix.acs.privilege.management.PrivilegeManagementService;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+@Test
+public class DefaultResourceAttributeStoreTest {
+    @Mock
+    private PrivilegeManagementService privilegeManagementService;
+
+    @Autowired
+    @InjectMocks
+    private DefaultResourceAttributeStore defaultResourceAttributeStore;
+
+    @BeforeMethod
+    public void beforeMethod() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
```

Comment:
pls add a test for existing resource as well.  (and subject for DefaultSubject store)